docs/api/clipboard.md

```diff
@@ -1,7 +1,6 @@
 # clipboard
 
-The `clipboard` provides methods to perform copy and paste operations. The following example
-shows how to write a string to the clipboard:
+The `clipboard` object provides methods to perform copy and paste operations. The following example shows how to write a string to the clipboard:
```

Comment:
I realized in this set of docs that sometimes we refer to these as _modules_ and in some of the previous docs I've gone through we say _objects_ instead. We could call everything objects of course but modules makes sense too. @zcbenz @kevinsawicki what do you think?